src/EventSubscriber/ApiKeyVersionSubscriber.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+
+class ApiKeyVersionSubscriber implements EventSubscriberInterface
+{
+    private string $apiVersionKey;
+
+    public function __construct(string $apiVersionKey)
+    {
+        $this->apiVersionKey = $apiVersionKey;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            ResponseEvent::class => 'addApiVersionKey'
+        ];
+    }
+
+    public function addApiVersionKey(ResponseEvent $event): void
+    {
+        if (\strpos($event->getRequest()->attributes->get('_route'), 'api_') !== false) {
```

Comment:
This can throw a fatal error because `$event->getRequest()->attributes->get('_route')` can return null.